code-challenges.test.js

```diff
@@ -32,8 +73,47 @@ const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+describe("remaindersOnly", () => {
+  it("returns an array of only the REMAINDERS of the numbers when divided by 3", () => {
+    const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
+    expect(remaindersOnly(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
+    expect(remaindersOnly(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+
+//  ReferenceError: remaindersOnly is not defined
+
 // b) Create the function that makes the test pass.
 
+// Pseudo Code:
+  // Create a function named remaindersOnly that takes in an array as an argument
+  // Input: Array of mixed data, parameter: array
+  // Iterate over the array using HOF .map and set to a variable called remainders
+  // Use conditional state if to find values in the array that have the data type number using typeof method
+  // For all numbers, apply % 3 to find the remainders after dividing by 3 
+  // Use HOF .filter on the mapped array called remainders and only return the remainder values using typeof method again since the original array had mixed data types
+  // Output: Array of only the remainders of the numbers when divided by 3
+
+const remaindersOnly = (array) => {
+  const remainders = array.map(value => {
+    if(typeof value === "number"){
+      return value % 3
+    }
+  })
+  return remainders.filter(value => typeof value === "number")
+}
+
+
+console.log(remaindersOnly(hodgepodge1)) // ->    [ 2, 0, -1, 0 ]
+console.log(remaindersOnly(hodgepodge2)) // -> [ 2, 1, -1 ]
```

Comment:
Logic looks excellent!  Just remember, you don't need to console log since your function is being invoked in your test.